README.md

```diff
@@ -150,16 +150,35 @@ If you have new features or analysis you'd like to add to tracing, feel free to
 
 Deploying a contract:
 
-```
+```bash
 npx saddle deploy MyContract Arg0 Arg1 -n rinkeby
 ```
 
 Verifying a contract on [Etherscan](https://etherscan.io):
 
-```
+```bash
 npx saddle verify "{Etherscan API Key}" MyContract Arg0 Arg1 -n rinkeby
 ```
 
+You can also start a saddle console:
+
+```bash
+npx -n --experimental-repl-awaitsaddle console
+```
+
+```javascript
+Saddle console on network development Web3ProviderEngine
+> saddle
+Saddle {
+  ...
+}
+> .deploy Counter
```

Comment:
Is the command `.deploy` or `deploy`?

Reply:
.deploy, it's a `dot` command. you could also do `saddle.deployContract(...)` if you want to spell it out in code